server/routes/private.js

```diff
@@ -145,5 +145,24 @@ router.get('/test', async (req, res) => {
     res.send(true);
 });
 
-module.exports = router
+router.post('/get-workers-for-school', async (req, res) => {
+    // Validate appropriate parameters are passed to view workers at each school 
+    const getWorkersForSchool = Joi.object({
+        schoolId: Joi.number().integer().required(), //The student must specify their school ID in order to view the workers
+    });
+
+    const query = req.query ? req.query : {};
+
+    const schoolId = query.schoolId ? query.schoolId : null;
+
+    const { error } = getWorkersForSchool.validate({schoolId});
+
+    if (!_.isNil(error)) res.send(error);
+
+    // Attempts to fetch all workers for a specific school 
+    const workerIds = await schoolHandler.getWorkerIdsForSchool(schoolId);
```

Comment:
Hi Simran,
I would recommend importing the schoolHandler at the top of the file similar to the other handler imports.

Reply:
Hi Amy! Thank you I have added in the handler import!